content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,200 @@
+
+The API Gateway menu provides the following operations:
+
+1. Create and manage API client credential association in order to obtain:
+
+   1. Instance ID of the DSCC at the particular region (Hexadecimals).
+   2. Client ID (Hexadecimals).
+   3. Client Secret (Hexadecimals).
+   4. URL to the HPE GreenLake Console API end-point (string).
+   5. URL to the HPE DSCC API end-point (string).
+2. Generate Access Token, change Client Secret, and delete Access Token.
+
+![API Gateway](/img/api-gateway-block.png "DSCC API Gateway")
+
+### Manage API Client Applications
+
+Each instance of the API client credential represents the authorization relationship between the client application and the DSCC REST API end point. Click on the Create Credentials button to generate a client credential. Afterward the user can obtain the client secret and client secret, and used them to generate the access token.
+
+![](/img/create-credentials-button.png)
+
+Inside the Create Credentials menu, click on the V button to show the pull down list and use the mouse to click on the desired application. For the purpose of using the DSCC REST API, please select Data Services Cloud Console instance in the correct region where the array has been deployed. The list shows all instances with the region where the applications are deployed.
+
+![select the application](/img/select-the-desired-application.png "Choose application (DSCC)")
+
+After selecting the correct application, enter the Credential Name (Please see HPE GreenLake Cloud Console User Guide for supported characters).  Click the Create Credentials button to proceed with Client Credentials creation.
+
+![Create Credential no input yet](/img/create-credentials-menu.png "Generate Client Credentials 1st time")
+
+Once the Create Credential button is clicked, the following information about the OAuth (Open Authorization) will be shown. These are the Client ID and Client Secret information menu that is going to be shown only once. Please copy both Client ID and Client Secret to a safely recorded location. In the case that the user missed to copy these information, then the user can only regenerate the Client Secret at later time using the Client-Secret Reset menu.
+
+![](/img/api-client-credential-created.png "Credentials Created Close")
+
+After closing the credential creation menu, the user can observe the prior created API Client Credentials by identifying the Credential Name.
+
+![](/img/application-credential-created-prior-shown.png "API Client Credentials are created")
+
+Clicking on the down arrow, user can generate the access token required to use the API.
+
+![](/img/api-client-credential-get-access-token.png "Time to obtain the Access Token")
+
+After clicking on the Generate Access Tokens button, the menu requires user to enter the Client Secret obtained from the API Client Credentials. This is the time to copy and paste the client secret from Credential Created menu so that the user can obtain the access token.
```

Comment:
Add "the" prior to "user to enter the Client Secret"
In the last line, change "client secret" to "Client Secret" for consistency